src/misc/msccl/msccl_setup.cc

```diff
@@ -69,7 +69,7 @@ ncclResult_t mscclSetupScratch(struct mscclAlgo* hostAlgo, hipStream_t stream) {
   size_t sizeNeeded = (status.nBytes * (size_t)(hostAlgo->nScratchChunks)) / (size_t)(hostAlgo->nChunksPerLoop);
   if (sizeNeeded > status.scratchBufferSize){
     NCCLCHECK(ncclCudaFree(status.scratchBuffer));
-    NCCLCHECK(ncclCudaCalloc((char**)&status.scratchBuffer, sizeNeeded));
+    NCCLCHECK(ncclCudaCalloc((char**)&status.scratchBuffer, sizeNeeded, stream, true));
```

Comment:
From experiences learned in #967, it's better to use `ncclCudaMalloc` to cope with graph capture, as there are `hipStreamSynchronize` calls inside `ncclCudaCalloc`.

Reply:
done